app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :likes
+  has_many :comments
```

Comment:
- [ ] In order to complete the relationship, we can do something like this below:
```
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```

Reply:
Hi @elisha2kyakpo1 :wave:, could you please explain what `foreign key: 'author_id` does?

Reply:
Hi @laiifuu , here `foreign key: 'author_id` is used to resemble user_id.